catalog/UMT/UMT_Climate_IrrMapper_RF_v1_1.jsonnet

```diff
@@ -98,6 +101,10 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
     IrrMapper: A Machine Learning Approach for High Resolution Mapping of
     Irrigated Agriculture Across the Western U.S.,
     Remote Sens. 2020, 12, 2328. [doi:10.3390/rs12142328](https://doi.org/10.3390/rs12142328)
+
+    Ketchum, D., Hoylman, Z.H., Huntington, J. et al. Irrigation intensification impacts sustainability
+    of streamflow in the Western United States. Commun Earth Environ 4, 479 (2023). [doi:10.1038/s43247-023-01152-2](https://doi.org/10.1038/s43247-023-01152-2)
+    
   |||,
   'gee:terms_of_use': ee.gee_terms_of_use(license),
   'gee:user_uploaded': true,
```

Comment:
could you also add the new usage and preview scripts in https://github.com/google/earthengine-catalog/tree/main/examples/UMT ?